pkg/kubernetes/deployer.go

```diff
@@ -0,0 +1,103 @@
+package kubernetes
+
+import (
+	"fmt"
+	"time"
+
+	apps "k8s.io/api/apps/v1"
+	core "k8s.io/api/core/v1"
```

Comment:
in my opinion, I think it would be better to specify the group and version together. like appsv1, corev1

Reply:
I updated!

Reply:
I updated!